### PR TITLE
Improvements I think

### PR DESCRIPTION
**Pull request by HitaloM**
**About:**
- Heroku support has been removed
- Fixes/improvements in the code
- Simplified code
- Linting with flake8
- Code formatted with black
- Imports reorganized with isort (using black profile)
- Added SPDX-License-Identifier
- Disabled the handler check for plug-ins (not working properly)

### CODE REVIEW

**Review by AndrielFR — APPROVED** [review]
hitalo aguenta

File: userlixo/handlers/bot/callback_query/info_plugin.py [hunk 1]
```diff
@@ -60,11 +64,10 @@ async def on_switch_plugin(c, cq):
     basename = cq.matches[0]["basename"]
     plugin_type = cq.matches[0]["plugin_type"]
     deactivate = cq.matches[0]["deactivate"]
+    plugin = plugins[plugin_type][basename]
     if basename not in plugins[plugin_type]:
         return await cq.answer(lang.plugin_not_found(name=plugin))
```
Comment (usernein) [inline userlixo/handlers/bot/callback_query/info_plugin.py]:
```suggestion

    if basename not in plugins[plugin_type]:
        return await cq.answer(lang.plugin_not_found(name=basename))
    plugin = plugins[plugin_type][basename]
```

File: userlixo/login.py [hunk 2]
```diff
@@ -110,7 +110,6 @@ async def main():
         print(f"[blue]{session_string}[/]\n")
         return await user.stop()
```
Comment (usernein) [inline userlixo/login.py]:
```suggestion

    login_bot = True
```

File: userlixo/login.py [hunk 3]
```diff
@@ -121,15 +120,13 @@ async def main():
             f"[bold yellow]I found an existing session from bot [/][cyan]{mention}[/][bold yellow]. Do you want to use it? [/][cyan]\[yn]",
             end="",
         )
-        c = click.getchar(True)
-        login_bot = c == "n"
+        click.getchar(True)
```
Comment (usernein) [inline userlixo/login.py]:
```suggestion
        c = click.getchar(True)
        login_bot = c == "n"
```

File: userlixo/login.py [hunk 4]
```diff
@@ -121,15 +120,13 @@ async def main():
...
 
     print("\n[bold green]- Logging in the assistant bot...")
-    if login_user:
-        if os.path.exists("bot.session"):
-            os.remove("bot.session")
+    if login_user and os.path.exists("bot.session"):
```
Comment (usernein) [inline userlixo/login.py]:
```suggestion
    if login_bot and os.path.exists("bot.session"):
```

File: userlixo/config.py [hunk 5]
```diff
@@ -61,15 +58,11 @@ async def load_env():
...
-                await Config.create(key=env_key, value=value_on_env)
-            elif env_key in restricted_vars:
-                os.environ[env_key] = value_on_env
-            else:
-                missing_vars.append([env_key, value_on_env, env_info])
+        if not value_on_db and env_key in restricted_vars:
+            os.environ[env_key] = value_on_env
             continue
+        elif not value_on_db:
+            missing_vars.append([env_key, value_on_env, env_info])
```
Comment (usernein) [inline userlixo/config.py]:
```suggestion
        if not value_on_db:
            if env_key in restricted_vars:
                os.environ[env_key] = value_on_env
            else:
                missing_vars.append([env_key, value_on_env, env_info])
            continue
```

**Review by usernein — CHANGES_REQUESTED** [review]
hitalo aguenta

**Review by usernein — APPROVED** [review]
hitalo parabéns

**Comment by HitaloM** [comment]
> hitalo parabéns

Obrigado 😊

**Review by alissonlauffer — APPROVED** [review]
Fix the conflicts, thanks 👍

**Comment by AndrielFR** [comment]
nice bro.

**Comment by HitaloM** [comment]
> Fix the conflicts, thanks 👍

Larga de ser loco

**Comment by HitaloM** [comment]
Alô

**Review by AndrielFR — APPROVED** [review]
nice. 👍

**Comment by alissonlauffer** [comment]
What about https://t.me/SpamTherapy/212931?

**Comment by HitaloM** [comment]
> What about https://t.me/SpamTherapy/212931?

It would be a backup branch, just @usernein create before merging this PR.

**Pull request merged**